apps/server/package.json

```
@@ -24,6 +24,7 @@
   "dependencies": {
     "cors": "2.8.5",
     "express": "4.19.2",
+    "express-expeditious": "^5.1.1",
```

``
```suggestion
    "express-expeditious": "5.1.1",
```

Eliminar las carets